docs/guides/install/index.html

```diff
@@ -3,215 +3,248 @@
-              elAdapter.parentElement.removeChild(elAdapter);
-            }
-          </script>
-          <script src="path/to/custom-elements-es5-adapter.js"></script>
-        </span>
-
-        <!-- HelixUI custom element definitions -->
-        <script src="cdn/path/to/helix-ui.js"></script>
-
-        <!-- load your application logic here -->
-        <script src="domain/path/to/your/app.js"></script>
-      </body>
-    {% endcode %}
+    <section><!-- TL;DR -->
+      <h2 class="hxSectionTitle">TL;DR</h2>
+      {% code 'html' %}
+        <!DOCTYPE html>
+        <html lang="en">
+          <head>
+            <!-- Head stuff goes here -->
+
+            <link rel="stylesheet" href="node_modules/helix-ui/dist/styles/helix-ui.css" />
+
+            <script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+            <!-- polyfills injected here (if needed) -->
+          </head>
+          <body>
+            <!-- Application markup goes here -->
+
+            <!-- intelligently load ES5 Adapter (if needed) -->
+            <span id="ce-es5-adapter">
+              <script>
+                if (!window.customElements) {
+                  var elAdapter = document.querySelector('#ce-es5-adapter');
+                  elAdapter.parentElement.removeChild(elAdapter);
+                }
+              </script>
+              <script src="node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+            </span>
+
+            <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+            <script src="node_modules/helix-ui/dist/helix-ui.browser.js"></script>
+            <script src="path/to/your/app.js"></script>
+          </body>
+        </html>
+      {% endcode %}
+    </section>
+
+    <section><!-- Polyfills -->
+      <h2 class="hxSectionTitle">Polyfills</h2>
+      <p>
+        HelixUI requires two polyfills to ensure maximum compatibility.
+      </p>
+    </section>
+
+    <section><!-- polyfill.io -->
+      <h2 class="hxSectionTitle">polyfill.io</h2>
+      <p>
+        The first polyfill is an online service that uses browser detection to
+        serve the smallest set of polyfills needed to accommodate missing functionality
+        (<a href="https://polyfill.io">polyfill.io</a>).
+      </p>
+      <p>
+        Load the script before HelixUI and any application logic.
+      </p>
+      {% code 'html' %}
+        <body>
+          ...
+
+          <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+
+          <!-- HelixUI custom element definitions go here -->
+
+          <!-- Application logic goes here -->
+        </body>
+      {% endcode %}
+    </section>
+
+    <section><!-- webcomponentsjs -->
+      <h2 class="hxSectionTitle">webcomponentsjs</h2>
+      <section>
+        <p>
+          The second polyfill adds support for web component technologies, using
```

Comment:
Consider rewording to "The second polyfill uses two scripts that add support for web component technologies."